### DISCUSSION TITLE
Can we do something so that this server becomes vanilla??

### DISCUSSION THREAD

**Discussion by JustaRandomGuy02** [thecoder-001/MineColab#41] [Q&A] [answered]
i built a huge farm and now it doesnt workk due to paper, so is there anyway i can change this server to vanilla??

**Comment by thecoder-001** [comment, marked as answer]
Hello!
Yup, you can. It has been already answered [here](https://github.com/thecoder-001/MineColab/discussions/34#discussioncomment-691362) for fabric, but the same steps would apply for vanilla. You would also need to move your world folder around a bit, since paper stores other dimensions (nether & end) a bit differently, you can find many guides online (search for keywords: world migration from paper to vanilla).
Feel free to post if that didn't work out for you.
ps: don't forget to backup your world before performing any of the steps, just in case.
Cheers!

**Reply by JustaRandomGuy02** [reply]
I did it, it works totally fine. Thank you!
Just one question, Do i need to change the run command  ( !java -Xmx6144M -Xm... ) for vanilla?
